buildscripts/Build.proj

```diff
@@ -207,24 +221,37 @@ limitations under the License.
 
 	</Target>
 
-
 	<Target
-		Name="_ExecStatLight"
+		Name="_ExecNUnit_Without_IntegrationTest"
```

Comment:
Glad to hear it! This was just to get it going. I have a feeling we would need to refine this anyway once we start embedding dotnet core elements. So we will be revisiting this.